centralledger/values.yaml

```diff
@@ -74,9 +74,9 @@ containers:
 
 config:
   # db_type can either be 'postgres' or 'mysql'. Ensure the correct DB is enabled and configured below: postgresql.enabled or mysql.enabled
-  db_type: 'postgres'
```

Comment:
Have you raised a bug for the helm repo that the postgres and mysql for central ledger need to be both enabled?